std_msgs/Header header
vision_msgs/Detection2DArray detections
sensor_msgs/Image[] masks
sensor_msgs/Image[] orig_image
string[] class_names   # Array to hold class names corresponding to detections
